llvm/lib/Transforms/IPO/FunctionSpecialization.cpp

```diff
@@ -641,12 +641,17 @@ bool FunctionSpecializer::run() {
         Metrics.analyzeBasicBlock(&BB, GetTTI(F), EphValues);
     }
 
+    // When specializing literal constants is enabled, always require functions
+    // to be larger than MinFunctionSize, to prevent excessive specialization.
+    bool RequireMinSize =
```

Comment:
May add "const" for a little bit of extra self-documentation. Or not.

Reply:
Good point - done :)